BringgSDK.js

```diff
@@ -668,6 +675,30 @@ var BringgSDK = (function () {
     watchingOrder = isWatching;
   };
 
+  module._setUpConfigByToken = function(developerToken) {
+    // Check if this new token with region
+    if(!developerToken || developerToken.indexOf("_") === -1) {
+      log('_setUpConfigByToken: developer access token doesnt contain region');
+      return;
+    }
+
+
+    // Extract the region
+    var tokenParts = developerToken.split('_');
+    var region = tokenParts[0],
+        token = tokenParts[1];
```

Comment:
you will crash if some dev decided to troll you and pass only "eu_" or "_". 
input validation with relevant error logs